src/api/config/routes/api_routes.rb

```diff
@@ -0,0 +1,408 @@
+    delete 'person/:login/token/:id' => 'person/token#delete', constraints: cons
+
+    # FIXME3.0: this is no clean namespace, a person "register" or "changepasswd" could exist ...
+    #           remove these for OBS 3.0
+    match 'person/register' => 'person#register', via: [:post, :put] # use /person?cmd=register POST instead
+    match 'person/changepasswd' => 'person#change_my_password', via: [:post, :put] # use /person/:login?cmd=changepassword POST instead
+    get 'person/:login/group' => 'person#grouplist', constraints: cons # Use /group?person=:login GET instead
+    # /FIXME3.0
+    match 'person/:login' => 'person#get_userinfo', constraints: cons, via: [:get]
+    match 'person/:login' => 'person#put_userinfo', constraints: cons, via: [:put]
+    match 'person/:login' => 'person#post_userinfo', constraints: cons, via: [:post]
+
+    ### /group
+    controller :group do
+      get 'group' => :index
+      get 'group/:title' => :show, constraints: cons
+      delete 'group/:title' => :delete, constraints: cons
+      put 'group/:title' => :update, constraints: cons
+      post 'group/:title' => :command, constraints: cons
+    end
+
+    ### /service
+    get 'service' => 'service#index'
+    get 'service/:service' => 'service#index_service', constraints: cons
+
+    ### /source
+
+    get 'source/:project/:package/_tags' => 'tag#package_tags', constraints: cons
+    get 'source/:project/_tags' => 'tag#project_tags', constraints: cons
+    get 'source/:project/_keyinfo' => 'source/key_info#show', constraints: cons
+
+    resources :about, only: :index
+
+    controller :attribute_namespace do
+      get 'attribute' => :index
+      get 'attribute/:namespace' => :index
+      # FIXME3.0: drop the POST and DELETE here
+      get 'attribute/:namespace/_meta' => :show
+      delete 'attribute/:namespace/_meta' => :delete
+      delete 'attribute/:namespace' => :delete
+      match 'attribute/:namespace/_meta' => :update, via: [:post, :put]
+    end
+
+    controller :attribute do
+      get 'attribute/:namespace/:name/_meta' => :show
+      delete 'attribute/:namespace/:name/_meta' => :delete
+      delete 'attribute/:namespace/:name' => :delete
+      match 'attribute/:namespace/:name/_meta' => :update, via: [:post, :put]
+    end
+
+    ### /architecture
+    resources :architectures, only: [:index, :show, :update] # create,delete currently disabled
+
+    ### /trigger
+    post 'trigger/rebuild' => 'trigger#rebuild'
+    post 'trigger/release' => 'trigger#release'
+    post 'trigger/runservice' => 'trigger#runservice'
+    post 'trigger/webhook' => 'services/webhooks#create'
+
+    ### /issue_trackers
+    get 'issue_trackers/issues_in' => 'issue_trackers#issues_in'
+    resources :issue_trackers, only: [:index, :show, :create, :update, :destroy] do
+      resources :issues, only: [:show] # Nested route
+    end
+
+    ### /tag
+
+    # routes for tagging support
+    #
+    # get 'tag/_all' => 'tag',
```

Comment:
Should we remove those commented routes? I guess it's fine to do it.

Reply:
I would remove it.

Reply:
Yes, I'm fine with this!